### PR TITLE
Add rating & Grid comp.

### PR DESCRIPTION
**Pull request by VLeads**
Added Grid component
Add Rating component

Done minor CSS fixes

### CODE REVIEW

**Review by SukanyaSen530 — APPROVED** [review]
All look's good, you can go ahead 👍🏼

File: styles/lib/_rating.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.ratings input:checked ~ label:before {
+    color: #f8dd43; 
+  }
+
+.ratings input:checked ~ label:hover:before,
+.ratings input:checked ~ label:hover ~ label:before {
+    color: #dfbf09; 
+}
```
Comment (Maianki) [inline styles/lib/_rating.css]: Use CSS variable for colors

**Review by Maianki — APPROVED** [review]
Suggested a few changes everything else looks good.

File: styles/lib/_rating.css (hunk 1 above)
Comment (VLeads) [reply styles/lib/_rating.css]: Done

**Pull request merged**